code-security/admin_guide/scan-monitor/iac-tag-and-trace.adoc

```diff
@@ -0,0 +1,64 @@
+
+== IaC Tag and Trace
+
+Prisma Cloud Code Security supports infrastructure-as-code (IaC) tags that help you trace the link for resources that are deployed from code to your cloud infrastructure. 
+Using Yor, an open-source auto-tagging tool that supports Terraform, CloudFormation, and Serverless, you can adds tags to all resource blocks in your repository. 
+Yor uses YAML configuration to tag and trace resources, and these automated tags are unique to a repository. 
+Yor automatically creates additional tags that contain details such as:
+
+* Git organization, repository, and the exact file that contains the infrastructure-as-code (IaC) template used to create the cloud resource.
```

Comment:
Should you include a link to Supported Git Tags here? https://github.com/bridgecrewio/yor/blob/main/docs/1.Welcome/welcome.md#supported-git-tags